crates/runtime/src/core_lib/koto.rs

```diff
@@ -51,5 +55,87 @@ pub fn make_module() -> KMap {
         unexpected => type_error_with_slice("a single argument", unexpected),
     });
 
+    result.add_fn("load", |ctx| match ctx.args() {
+        [KValue::Str(s)] => try_load_koto_script(ctx, s),
+        [KValue::Object(o)] if o.is_a::<File>() => {
+            let mut file = o.cast_mut::<File>().unwrap();
+            let contents = file.read_to_kstring()?;
```

Comment:
As far as I can tell you don't need a `KString` here, the compiler just wants `&str`, and `KotoFile` returns a `String` from `read_to_string`, so maybe something like a getter for the inner `Ptr<KotoFile>` would do the trick? 
e.g. `let contents = file.inner().read_to_string()?;`?